packages/d3fc-axis/src/axis.js

```diff
@@ -13,6 +13,8 @@ const axis = (orient, scale) => {
     let tickSizeOuter = 6;
     let tickSizeInner = 6;
     let tickPadding = 3;
+    let centerAlignTicks = false;
```

Comment:
I'm not sure about the naming. The convention seems to be a `tick` prefix - maybe we could we go for `tickCenter`/`tickAlign` (with `'center'` as a value)?